packages/ui/src/components/ListV2/HeaderRow.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled'
+import type { ReactNode } from 'react'
+import { Checkbox } from '../Checkbox'
+import { HeaderCell } from './HeaderCell'
+import { useListContext } from './ListContext'
+
+const StyledHeaderRow = styled.div`
+  // List itself also apply style about common templating between HeaderRow and other Rows
```

Comment:
Not sure if `//` comments work inside emotion?
```suggestion
  /* List itself also apply style about common templating between HeaderRow and other Rows */
```